Makefile

```diff
@@ -70,7 +70,7 @@ windows-local:
 	mkdir -p bin
 	GOOS=windows GOARCH=amd64 go build -ldflags "-X main.vendorVersion=${VERSION}" -o bin/gcfs-csi-driver.exe ./cmd/
 
-build-image-and-push: image
+build-image-and-push: image init-buildx
```

Comment:
why is this init-buildx and not init-build?

Reply:
Fixed